### PR TITLE
CHK-1377-apim-configuration-for-openapi

### PR DESCRIPTION
**Pull request by Fernando-Granato**
### List of changes
Added new apim configuration wallet.
- New new base_policy file
- New openapi.json file

### Motivation and context
Added the configuration for the new pagopa-wallet API group.
The apim path for the api is **https://api.dev.platform.pagopa.it/wallet-service/v1/wallets**

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

**Review by infantesimone — APPROVED** [review]

File: src/domains/wallet-app/04_apim_wallet.tf [hunk 1]
```diff
@@ -19,3 +19,52 @@ module "apim_wallet_product" {
...
+  resource_group_name = local.pagopa_apim_rg
+  api_management_name = local.pagopa_apim_name
+  display_name        = local.apim_wallet_service_api.display_name
+  versioning_scheme   = "Segment"
+}
+
+module "apim_wallet_service_api_v1" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//api_management_api?ref=v6.3.0"
+
+  name                  = format("%s-wallet-service-api", local.project)
```
Comment (infantesimone) [inline src/domains/wallet-app/04_apim_wallet.tf:48]: mmm local.project has `wallet`, so the final name is `pagopa-d-weu-wallet-wallet-service-api`

Comment (infantesimone) [inline src/domains/wallet-app/04_apim_wallet.tf:48]: mmm local.project has `wallet`, so the final name is `pagopa-d-weu-wallet-wallet-service-api`

Comment (Fernando-Granato) [reply src/domains/wallet-app/04_apim_wallet.tf]: I changed the name by removing the **wallet** prefix

Comment (Fernando-Granato) [reply src/domains/wallet-app/04_apim_wallet.tf]: I changed the name by removing the **wallet** prefix

**Review by infantesimone — APPROVED** [review]

**Pull request merged**